pkg/entity/flag.go

```diff
@@ -15,15 +15,17 @@ import (
 type Flag struct {
 	gorm.Model
 
-	Key                string `gorm:"type:varchar(64);unique_index:idx_flag_key"`
-	Description        string `sql:"type:text"`
-	CreatedBy          string
-	UpdatedBy          string
-	Enabled            bool
-	Segments           []Segment
-	Variants           []Variant
+	Key         string `gorm:"type:varchar(64);unique_index:idx_flag_key"`
+	Description string `sql:"type:text"`
+	CreatedBy   string
+	UpdatedBy   string
+	Enabled     bool
+	Segments    []Segment
+	Variants    []Variant
+	SnapshotID  uint `json:"-"`
+
 	DataRecordsEnabled bool
-	SnapshotID         uint `json:"-"`
```

Comment:
What was this field used for?

Reply:
Oh, I didn't remove it, just moved it a little bit above. It was used to store the FlagSnapshotID